torch/csrc/lazy/backend/backend_device.cpp

```diff
@@ -11,7 +13,7 @@ namespace lazy {
 BackendDevice::BackendDevice()
   : type_(std::make_shared<BackendDeviceType>()) {}
 
-BackendDevice::BackendDevice(std::shared_ptr<BackendDeviceType>&& type, int ordinal)
+BackendDevice::BackendDevice(std::shared_ptr<BackendDeviceType>&& type, int64_t ordinal)
```

Comment:
Nit: any specific reason we want to move to int64 here?

Reply:
It's usually better to specify the length of the int. Usually, just use int64_t given we are now in the age of 64 bit.